data/src/entity_manager/system.py

```
@@ -3952,8 +3963,8 @@ def _verify_result(self, entity_class, id_value, cursor):
         # the length is calculated otherwise a tuple must created from the
         # single value
         id_value_type = type(id_value)
-        id_length = id_value_type == tuple and len(id_value) or 1
-        id_value = id_value_type == tuple and id_value or (id_value,)
+        id_length = len(id_value) if id_value_type == tuple else 1
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)